README.md

```diff
@@ -58,6 +58,15 @@ installers for all supported operating systems.
     sudo apt-get install etcher-electron
     ```
 
+##### Uninstall
+
+```sh
+sudo apt-get remove etcher-electron
+sudo apt-key del D401AB61
```

Comment:
Nah, I meant we should keep the PR but we should probably just remove the `sudo apt-key del D401AB61` line?